jetstream/macos-background-tab-power-savings.toml

```diff
@@ -93,5 +90,41 @@ type = "histogram"
 
 [metrics.fx_tab_click_ms.statistics.bootstrap_mean]
 
-[metrics.fx_tab_switch_total_e10s_ms.statistics.bootstrap_mean]
+[metrics.fx_tab_switch_composite_e10s_ms]
+data_source = "main_filtered"
+
+select_expression = "{{agg_histogram_mean('payload.histograms.fx_tab_switch_composite_e10s_ms')}}"
+
+friendly_name = "Fx Tab Click Ms"
+description = "Firefox: Time in ms spent on switching tabs in response to a tab click."
+category = "performance"
+type = "histogram"
+
 [metrics.fx_tab_switch_composite_e10s_ms.statistics.bootstrap_mean]
+
+[metrics.fx_tab_switch_total_e10s_ms]
+data_source = "main_filtered"
+
+select_expression = "{{agg_histogram_mean('payload.histograms.fx_tab_switch_total_e10s_ms')}}"
+
+friendly_name = "Fx Tab Click Ms"
+description = "Firefox: Time in ms spent on switching tabs in response to a tab click."
+category = "performance"
+type = "histogram"
+
+[metrics.fx_tab_switch_total_e10s_ms.statistics.bootstrap_mean]
+
+
+[data_sources]
+[data_sources.main_filtered]
+from_expression = """(
+    SELECT
+        *,
+        DATE(submission_timestamp) AS submission_date,
+        environment.experiments
+    FROM `moz-fx-data-shared-prod.telemetry_stable.main_v5`
+    WHERE mozfun.map.get_key(environment.experiments, 'macos-background-tab-power-savings') IS NOT NULL
```

Comment:
filtering on the beta channel might speed up things even more here (afaik this experiment is running on beta)

Reply:
Can we do that automatically when building these queries? I believe channel should be in the Experimenter API

Reply:
it's a little more complicated. I opened https://github.com/mozilla/jetstream/issues/1827 a while ago which mentions this as well